deployments/dev/environment.py

```diff
@@ -49,6 +49,7 @@ def env() -> Mapping[str, Optional[str]]:
             f'AZUL_TDR_{catalog.upper()}_SOURCES': ','.join([
                 'tdr:broad-jade-dev-data:snapshot/hca_dev_20210208_snaptest2___20210208',
                 'tdr:broad-jade-dev-data:snapshot/hca_dev_20210211_snaptest3___20210211',
```

Comment:
These should be removed. I should have specified that we want just this one snapshot in the catalog.